packages/oslo-converter-stakeholders/lib/utils/ToJsonTransformer.ts

```diff
@@ -2,13 +2,15 @@ import type { TransformCallback } from 'stream';
 import { Transform } from 'stream';
 import { ContributorType } from '@oslo-converter-stakeholders/enums/ContributorType';
 
-export class ToJsonLdTransformer extends Transform {
+export class ToJsonTransformer extends Transform {
```

Comment:
Can you eloborate why you changed the name? Is it because we actually generate a JSON object and then add a `@context` to it so it becomes JSON-LD?

Reply:
Sure! I though it would be weird to have both the `ToJsonLdTransformer` and `ToJsonTransformer` or like a `ToJsonAndJsonLdTransformer`. Since both are technically JSON-like, I felt that the name `ToJsonTransformer` worked for both outputs!